micro-rdk/src/gen/proto.rpc.webrtc.v1.rs

```diff
@@ -272,14 +272,19 @@ pub struct AnswerRequestErrorStage {
     #[prost(message, optional, tag="1")]
     pub status: ::core::option::Option<super::super::super::super::google::rpc::Status>,
 }
+/// AnswerRequestHeartbeatStage is sent periodically to verify liveness of answerer.
```

Comment:
did you generate it with make buf?

Reply:
Yep! Lmk if that's not the correct method.